examples/deployment/pipecat-cloud-example/README.md

```diff
@@ -0,0 +1,191 @@
+
+## Getting Started
+
+### 1. Set up Python environment
+
+We recommend using a virtual environment to manage your Python dependencies.
+
+```bash
+# Create a virtual environment
+python -m venv venv
+
+# Activate it
+source venv/bin/activate  # On Windows: venv\Scripts\activate
+
+# Install dependencies
+pip install -r requirements.txt
+pip install pipecatcloud
+```
+
+### 2. Get the starter project
+
+Clone the starter project from GitHub:
+
+```bash
+git clone https://github.com/daily-co/pipecat-cloud-starter
+cd pipecat-cloud-starter
+```
+
+or use the Pipecat Cloud CLI to initialize a new project:
+
+```bash
+mkdir pipecat-cloud-starter && cd pipecat-cloud-starter
+pcc init
+```
+
+### 3. Authenticate with Pipecat Cloud
+
+```bash
+pcc auth login
+```
+
+### 4. Acquire required API keys
+
+This starter requires the following API keys:
+
+- **OpenAI API Key**: Get from [platform.openai.com/api-keys](https://platform.openai.com/api-keys)
+- **Cartesia API Key**: Get from [play.cartesia.ai/keys](https://play.cartesia.ai/keys)
+- **Daily API Key**: Automatically provided through your Pipecat Cloud account
+
+### 5. Configure to run locally (optional)
+
+You can test your agent locally before deploying to Pipecat Cloud:
+
+- `DAILY_API_KEY` value can be found at [https://pipecat.daily.co](https://pipecat.daily.co) Under the `Settings` menu of your agent, in the `Daily` tab.
+
+```bash
+# Set environment variables with your API keys
+export CARTESIA_API_KEY="your_cartesia_key"
+export DAILY_API_KEY="your_daily_key"
+export OPENAI_API_KEY="your_openai_key"
+LOCAL_RUN=1 python bot.py
+```
+
+## Deploy & Run
+
+### 1. Build and push your Docker image
+
+```bash
+# Build the image (targeting ARM architecture for cloud deployment)
+docker build --platform=linux/arm64 -t my-first-agent:latest .
+
+# Tag with your Docker username and version
+docker tag my-first-agent:latest your-username/my-first-agent:0.1
+
+# Push to Docker Hub
+docker push your-username/my-first-agent:0.1
+```
+
+### 2. Create a secret set for your API keys
+
+The starter project requires API keys for OpenAI and Cartesia:
+
+```bash
+# Copy the example env file
+cp env.example .env
+
+# Edit .env to add your API keys:
+# CARTESIA_API_KEY=your_cartesia_key
+# OPENAI_API_KEY=your_openai_key
+
+# Create a secret set from your .env file
+pcc secrets set my-first-agent-secrets --file .env
+```
+
+Alternatively, you can create secrets directly via CLI:
+
+```bash
+pcc secrets set my-first-agent-secrets \
+  CARTESIA_API_KEY=your_cartesia_key \
+  OPENAI_API_KEY=your_openai_key
+```
+
+### 3. Deploy to Pipecat Cloud
+
+```bash
+pcc deploy my-first-agent your-username/my-first-agent:0.1
```

Comment:
What happens if my image is private? Do I need to do anything different in this case?